assign-2/src/data/MenuItem.java

```diff
@@ -0,0 +1,16 @@
+package data;
+
+public abstract class MenuItem {
```

Comment:
Very nice choice to create an abstraction of a menu item! This class should not be inside the `data` layer. Move it to the `presentation` layer instead.

Reply:
I will move it to the presentation layer, Thank you!